AmericanMedicalAssociation/ama-style-guide-2
# [EWL-5306] SG2 | Fix Broken Molecule Twig Templates

## Ticket(s)

**Jira Ticket**
- [EWL-5306: SG2 | Fix Broken Molecule Twig Templates](https://issues.ama-assn.org/browse/EWL-5306)

## Description
Multiple bugs have been introduced into the `develop` branch of SG2 which has caused templates includes of various molecules to not render correctly.

This update corrects template include paths from previous molecule template reorganizing. All backstop reference image issues introduced from the reorganizing as been included in this PR.

Chromy errors were resolved with updates to backstop test cases that were not able to find the correct pattern lab pages.


## To Test
- Pull `bugfix/EWL-5306-broken-twig-templates` branch
- Run `gulp serve` in ama-style-guide-2/styleguide to ensure a local version of style guide is running.
- Run `backstop test` and confirm no errors occur.
- Visit all pages and organisms to confirm there are no errors with template include paths.

## Visual Regressions

Updates to reference images have been included in this PR.


## Relevant Screenshots/GIFs
N/A


## Remaining Tasks
N/A


## Additional Notes
N/A

---

[Guidelines for Contribution](CONTRIBUTING.md)
[SG2 Standards](ama-style-guide-2/docs/standards.md)
